<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FromToEdit</class>
 <widget class="QWidget" name="FromToEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>从</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="from_det"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="from_tet"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>到</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="to_det"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="to_tet"/>
   </item>
   <item>
    <widget class="QPushButton" name="apply_btn">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
